components/daisy/Progress.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Progress() {
```

Comment:
This will be always empty progress. Wdyt about adding props to this progress element?